odpf/optimus/core/v1beta1/job_spec.proto

```diff
@@ -47,6 +47,13 @@ service JobSpecificationService {
     };
   }
 
+  // GetJob read a job spec for provided filters
+  rpc GetJob(GetJobRequest) returns (GetJobResponse) {
```

Comment:
Also this one, should we also reuse the `GetJobSpecification` but making the project agnostic?

Reply:
Also @sravankorumilli , could you suggest on this? Because, depending on which approach we take, more refactors might be required.

Reply:
After discussion with Sravan, lets not touch the `GetJobSpecification`. We can use what @SumitAgrawal03071989 suggested, but can you consider these changes? @SumitAgrawal03071989 
- Rename it to `GetJobSpecifications`
- Path should be `jobs`
- Response should be `repeated`

Reply:
@arinda-arif Sure